### ISSUE TITLE
pdf2txt error in command line cannot match the files argument

### ISSUE THREAD

**Issue by udaykapur** [euske/pdfminer#252]
#164 This issue still remains unresolved on Win10 - python 3.7.1

Has someone found a solution yet?

**Comment by noopur2805** [comment]
Try replacing file() function with open() and write output file in bytes with 'wb'.

**Comment by AyeshaSarwar** [comment]
Hi
I ran into the same problem. Can you help me telling how did you resolve this issue.
Reference
https://github.com/euske/pdfminer/issues/164  
Thanks

**Comment by AyeshaSarwar** [comment]
it always says required argument files
 I am using this syntax
pdf2txt.py "D:\test.pdf"
also tried
pdf2txt.py -c utf-8 "D:\test.pdf"

**Comment by AyeshaSarwar** [comment]
python 3.8.1
windows10

**Comment by noopur2805** [comment]
> it always says required argument files
> I am using this syntax
> pdf2txt.py "D:\test.pdf"
> also tried
> pdf2txt.py -c utf-8 "D:\test.pdf"

Getting error even after the above mentioned changes?

**Comment by AyeshaSarwar** [comment]
Yes, it runs from administrative power shell but not from command prompt on windows

**Comment by AyeshaSarwar** [comment]
C:\Users\Ayesha>pdf2txt.py D:/test.pdf
usage: pdf2txt.py [-h] [--debug] [--disable-caching] [--page-numbers PAGE_NUMBERS [PAGE_NUMBERS ...]]
                  [--pagenos PAGENOS] [--maxpages MAXPAGES] [--password PASSWORD] [--rotation ROTATION]
                  [--no-laparams] [--detect-vertical] [--char-margin CHAR_MARGIN] [--word-margin WORD_MARGIN]
                  [--line-margin LINE_MARGIN] [--boxes-flow BOXES_FLOW] [--all-texts] [--outfile OUTFILE]
                  [--output_type OUTPUT_TYPE] [--codec CODEC] [--output-dir OUTPUT_DIR] [--layoutmode LAYOUTMODE]
                  [--scale SCALE] [--strip-control]
                  files [files ...]
pdf2txt.py: error: the following arguments are required: files

**Comment by AyeshaSarwar** [comment]
Power Shell Output

PS D:\EducAtiOn\Thesis\Tasks\2\Parsr> pdf2txt.py D:/simple1.pdf
Hello

World

Hello

World

H e l l o

W o r l d

H e l l o

W o r l d

**Comment by AyeshaSarwar** [comment]
It is strange but what do

**Comment by noopur2805** [comment]
sorry I'm unaware of what might be happening in such case :(

**Comment by noopur2805** [comment]
> It is strange but what do

sorry I'm unaware of what might be happening in such case :(
